src/v/kafka/server/handlers/details/security.h

```diff
@@ -411,4 +412,82 @@ authorized_operations(request_context& ctx, const T& resource) {
     return allowed_operations;
 }
 
+/*
+ * build resource pattern filter bits from delete acl request filter
+ */
+inline security::resource_pattern_filter
+to_resource_pattern_filter(const delete_acls_filter& filter) {
+    std::optional<security::resource_type> resource_type;
+    switch (filter.resource_type_filter) {
+    case 1:
+        // wildcard
+        break;
+    default:
+        resource_type = to_resource_type(filter.resource_type_filter);
+    }
+
+    std::optional<security::resource_pattern_filter::pattern_filter_type>
```

Comment:
unit test for this is needed since the matching type is complex.

Reply:
Will do this